src/items/unions.md

```diff
@@ -29,18 +29,23 @@ struct types, except that it must specify exactly one field:
 let u = MyUnion { f1: 1 };
 ```
 
-The expression above creates a value of type `MyUnion` with active field `f1`.
-Active field of a union can be accessed using the same syntax as struct fields:
+The expression above creates a value of type `MyUnion` and initializes the
+storage using field `f1`.  The union can be accessed using the same syntax as
```

Comment:
```suggestion
storage using field `f1`. The union can be accessed using the same syntax as
```

Reply:
Ah, my emacs and also my brain automatically use double-space between sentences.  I'll see if I can remove them all, but they might appear again as I edit: a double-space after a full stop is pretty deep in my muscle memory at this point. ;)